sagemaker-huggingface/text-summarizer/huggingface-custom-text-summarizer.ipynb

```diff
@@ -0,0 +1,2626 @@
+{
```

Comment:
<p>I see you're writing this to a file, but since it's such a long code block at 180 lines, consider including it as a separate .py file alongside the notebook. Either is fine, but it may be more readable to separate it from the notebook body.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>